apps/researcher/src/messages/nl/messages.json

```diff
@@ -299,20 +293,23 @@
     "title": "Voeg een verhaal toe",
     "description": "Verhaal",
     "descriptionSubTitle": "Hier kunt u uw eigen verhaal toevoegen aan de beschrijving van dit object.",
-    "community": "Community",
-    "communitySubTitle": "Selecteer de community waarvoor u spreekt.",
     "citation": "Bron",
-    "citationSubTitle": "Geef informatie op die uw verhaal ondersteunt.",
+    "citationSubTitle": "Geef informatie die uw verhaal ondersteunt.",
     "inLanguage": "Taal",
-    "languageSubTitle": "Selecteer een taal voor het verhaal. Begin met typen om een taal te selecteren.",
+    "languageSubTitle": "Selecteer de taal van het verhaal. Begin met typen om een taal te selecteren.",
     "successfullyAdded": "Uw verhaal is toegevoegd aan het object. Het kan even duren voordat het zichtbaar is.",
-    "serverError": "Er is iets misgegaan tijdens het maken van het verhaal. Probeer het later opnieuw.",
+    "labelAttributionId": "Persoonlijke profiel-URL",
+    "descriptionAttributionId": "Door uw verhaal toe te voegen, wordt uw persoonlijke URL opgeslagen bij het verhaal. Dit kan een link zijn naar uw persoonlijke LinkedIn, Facebook of ORCID-profiel. Het is mogelijk niet zichtbaar voor andere bezoekers, maar de gegevens worden opgeslagen in de cloud.",
+    "labelLicense": "Ik ga ermee akkoord dat alle toegevoegde informatie in de applicatie wordt gepubliceerd met een <link>CC-BY-4.0 licentie</link>",
+    "licenseLink": "https://creativecommons.org/licenses/by/4.0/deed.nl",
+    "serverError": "Er is iets misgegaan bij het aanmaken van het verhaal. Probeer het later opnieuw.",
     "buttonSubmit": "Voeg uw verhaal toe",
     "buttonCancel": "Annuleren",
-    "descriptionTooSmall": "Verhaal is vereist",
-    "communityIdTooSmall": "Community is vereist",
-    "citationTooSmall": "Bron is vereist",
-    "noCommunities": "U bent nog geen lid van een community die gemachtigd is een verhaal toe te voegen. U kunt een community maken of lid worden van een bestaande community.",
+    "descriptionRequired": "Verhaal is vereist",
+    "citationRequired": "Bron is vereist",
+    "invalidAttributionId": "Ongeldige persoonlijke profiel-URL",
```

Comment:
Line 302 refers to a 'persoonlijke URL'. Also use that term here, instead of 'persoonlijke profiel-URL'?